embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/features/UserFeaturesTest.kt

```diff
@@ -0,0 +1,62 @@
+package io.embrace.android.embracesdk.features
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import io.embrace.android.embracesdk.IntegrationTestRule
+import io.embrace.android.embracesdk.payload.SessionMessage
+import io.embrace.android.embracesdk.recordSession
+import org.junit.Assert.assertEquals
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+internal class UserFeaturesTest {
+
+    @Rule
+    @JvmField
+    val testRule: IntegrationTestRule = IntegrationTestRule(
+        harnessSupplier = {
+            IntegrationTestRule.newHarness(startImmediately = false)
+        }
+    )
+
+    @Test
+    fun `user info setting and clearing`() {
+        with(testRule) {
+            harness.androidServicesModule.preferencesService.userIdentifier = "customId"
+            harness.androidServicesModule.preferencesService.username = "customUserName"
+            harness.androidServicesModule.preferencesService.userEmailAddress = "custom@domain.com"
```

Comment:
nit: minor style preference for extracting a local variable of `preferencesService` here, or using `.apply {}`

Reply:
cool will do